playwright.config.js

```diff
@@ -1,7 +1,8 @@
 import { defineConfig } from '@playwright/test';
 
 export default defineConfig({
-  snapshotPathTemplate: './dist/playwright/{testFilePath}/{arg}{ext}',
+  fullyParallel: true,
+  snapshotPathTemplate: './dist/playwright/{arg}{ext}',
```

Comment:
It turned out to be simpler [just](https://github.com/CrowdStrike/glide-core/pull/727/files#diff-0f1d33468553eb92f9f0fa405732ffa15691246f11d05fbd6c8fc247da2c5734R16) add the test filename to the screenshot filename.